File: src/main/java/com/hyak4j/guess/GameFor.java
package com.hyak4j.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        /*int sum = 0;
        for (int i = 5; i >= 1; i--){
            sum += i;
            System.out.println("i: " + i + " sum: " + sum);
        }*/
        int secret = new Random().nextInt(10) + 1;
        Scanner scanner = new Scanner(System.in);
        for (int i = 1; i < 5; i++){
            System.out.println("Please enter a number: (" + i + "/4)");
            int number = scanner.nextInt();
            System.out.println("No" + i + ":" + number);
            if (number > secret){
                System.out.println("Lower");
            }else if (number < secret){
                System.out.println("Higher");
            }else {
                System.out.println("Bingo!");
                break;
            }
            if (i == 4){
                System.out.println("Game Over!");
                break;
            }
        }
        System.out.println("End");
    }
}

File: src/main/kotlin/com/hyak4j/kotlin/GameFor.kt
package com.hyak4j.kotlin

import java.util.Random
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val secret = Random().nextInt(10) + 1
    for (i in 1..4){
        println("Please enter a number (${i}/4): ")
        val number = scanner.nextInt()
        println("第${i}次$number")
        if (number == -1) break

        if (number > secret){
            println("Lower")
        }else if (number < secret){
            println("Higher")
        }else{
            println("Bingo!")
            break
        }
    }
    println("End")
}